### PR TITLE
Update part-two.rst - Spanish translation

### PR DESCRIPTION
**Pull request by jmanubarreiro**
Proposal for the spanish translation of part-two.rst

### CODE REVIEW

File: es/tutorials-and-examples/bookmarks/part-two.rst [hunk 1]
```diff
@@ -1,11 +1,456 @@
...
+            'conditions' => [
+                'Bookmarks.user_id' => $this->Auth->user('id'),
+            ]
+        ];
+        $this->set('bookmarks', $this->paginate($this->Bookmarks));
+        $this->set('_serialize', ['bookmarks']);
+    }
+
+
+Deberíamos actualizar también el método	``tags()`` y el método finder
```
Comment (markstory) [inline es/tutorials-and-examples/bookmarks/part-two.rst]: Lots of extra space here.

File: es/tutorials-and-examples/bookmarks/part-two.rst [hunk 2]
```diff
@@ -1,11 +1,456 @@
...
+relacionaodo, pero lo dejaremos como un ejercico para que lo hagas por tu cuenta.
+
+Mejorar la experiencia de etiquetado
+====================================
+
+Ahora mismo añadir nuevos tags es un proceso complicado desde que 
+``TagsController`` desautorizó todos los accesos.
+
+En vez de permitirlos podemos mejorar la *UI* para la selección de tags 
+utilizando un campo de texto separado por comas. Ésto proporcionará una mejor
```
Comment (albertcansado) [inline es/tutorials-and-examples/bookmarks/part-two.rst]: ``Esto``

File: es/tutorials-and-examples/bookmarks/part-two.rst [hunk 3]
```diff
@@ -1,11 +1,456 @@
...
+la función.
+
+Ahora puedes visitar ``/users/logout`` para desloguearte, deberías ser enviado 
+a la página de inicio.
+
+Habilitar registros
+===================
+
+Si no estás logueado e intentas acceder a **/users/add** eres reenviado a la 
+página de login. Deberíamos arreglar ésto si queremos permitir que la gente se
```
Comment (albertcansado) [inline es/tutorials-and-examples/bookmarks/part-two.rst]: ``esto``

File: es/tutorials-and-examples/bookmarks/part-two.rst [hunk 4]
```diff
@@ -1,11 +1,456 @@
...
+Tal vez quieras tomarte un tiempo para limpiar **Users/add.ctp** y eliminar los 
+enlaces erróneos o continuar con el siguiente apartado. No vamos a crear la 
+edición de usuarios, consulta o listado en este tutorial así que no funcionará 
+el control de ``AuthComponent`` para el acceso a esas acciones del controlador.
+
+Restringiendo el acceso a favoritos
+===================================
+
+Ahora que los usuarios pueden loguearse queremos restringir los favoritos que 
+uno puede ver a los que creó. Ésto lo haremos usando un adaptador de
```
Comment (albertcansado) [inline es/tutorials-and-examples/bookmarks/part-two.rst]: ``esto``

**Comment by markstory** [comment]
Can someone let me know when this is ready to merge :smile:

**Comment by jmanubarreiro** [comment]
I think it is

**Comment by jmanubarreiro** [comment]
By the way,

Muchas gracias @albertcansado por las correcciones.

**Pull request merged**